### DISCUSSION TITLE
Why is my GPU not being used?

### DISCUSSION THREAD

**Discussion by Cassiusor** [Jeffser/Alpaca#237] [Q&A]
This is my first time using a Local AI and Alpaca. Everything is working fine except I have one question. My GPU is not being used at all when the AI is giving me a output. Typically local AI models use the GPU for most of the processing so I'm a bit confused why it's only using my CPU. Why is that? I am running Fedora Silverblue, Alpaca Flatpak in the Gnome Store, and a AMD GPU if that helps.

**Comment by Jeffser** [comment]
Alpaca doesn't have full support for AMD GPUs due to the flatpak sandbox, if you need to run Ollama with GPU I recommend you run Ollama on a docker and just connect it to Alpaca using the 'connect to remote instance' feature available on the preferences dialog

**Reply by Cassiusor** [reply]
Will that work on a immutable distro like Fedora Silverblue?

**Reply by Jeffser** [reply]
Fedora Silverblue comes with Podman instead of docker, let me see if I can get a command for you to run

**Reply by Cassiusor** [reply]
Ok thanks!

**Reply by Jeffser** [reply]
Alright this command is going to create a folder named Ollama wherever you run it, you can't move that folder so please choose wisely where you want Ollama to save the models

podman run -d --gpus=all -v ollama:/root/.ollama -p 11434:11434 --name ollama ollama/ollama

I also recommend you this app called [Pods](https://flathub.org/apps/com.github.marhkb.Pods) to manage podman containers, they can be useful when using a system like silverblue
